includes/class-amp-story-post-type.php

```diff
@@ -407,10 +412,12 @@ public static function load_admin_fonts( $post ) {
 		if ( $post_story_data ) {
 			foreach ( $post_story_data as $page ) {
 				foreach ( $page['elements'] as $element ) {
-					$font = AMP_Fonts::get_font( $element['fontFamily'] );
-					if ( $font && ! in_array( $font['slug'], $font_slugs, true ) ) {
-						$fonts[]      = $font;
-						$font_slugs[] = $font['slug'];
+					if ( isset( $element['fontFamily'] ) ) {
```

Comment:
Could we do `continue` if `! isset( $element['fontFamily'] )` instead? To reduce nesting and improve readability.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/4122/commits/b7aa6d26c7839c7134782ecd0113fd9a66333697